bigtable-client-core-parent/bigtable-hbase/pom.xml

```diff
@@ -119,6 +119,26 @@ limitations under the License.
       <version>${commons-logging.version}</version>
     </dependency>
 
+    <!-- For veneer client settings configurations -->
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-core</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.api</groupId>
+      <artifactId>gax-grpc</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>com.google.auth</groupId>
+      <artifactId>google-auth-library-oauth2-http</artifactId>
+      <exclusions>
+        <exclusion>
+          <groupId>com.google.http-client</groupId>
+          <artifactId>google-http-client-jackson2</artifactId>
+        </exclusion>
+      </exclusions>
+    </dependency>
```

Comment:
@igorbernstein2 
These three dependencies needed to be declared. It seems veneer client settings are using classes from these three. I hope this should be fine. WDYT?

Reply:
yes, but why the exclude?

Reply:
This is due to a conflict happens in `bigtable-hbase-1.x-benchmark` module between google-http-client-jackson2's 1.34.0 & 1.34.2 versions.
```sh
WARNING] Rule 2: org.apache.maven.plugins.enforcer.RequireUpperBoundDeps failed with message:
Failed while enforcing RequireUpperBoundDeps. The error(s) are [
Require upper bound dependencies error for com.google.http-client:google-http-client-jackson2:1.34.0 paths to dependency are:
+-com.google.cloud.bigtable:bigtable-hbase-1.x-benchmarks:1.14.1-SNAPSHOT
  +-com.google.cloud.bigtable:bigtable-hbase:1.14.1-SNAPSHOT
    +-com.google.auth:google-auth-library-oauth2-http:0.20.0
      +-com.google.http-client:google-http-client-jackson2:1.34.0
and
+-com.google.cloud.bigtable:bigtable-hbase-1.x-benchmarks:1.14.1-SNAPSHOT
  +-com.google.cloud.bigtable:bigtable-hbase-1.x:1.14.1-SNAPSHOT
    +-com.google.cloud.bigtable:bigtable-client-core:1.14.1-SNAPSHOT
      +-com.google.cloud:google-cloud-core:1.93.1
        +-com.google.http-client:google-http-client-jackson2:1.34.2
and
+-com.google.cloud.bigtable:bigtable-hbase-1.x-benchmarks:1.14.1-SNAPSHOT
  +-com.google.cloud.bigtable:bigtable-hbase-1.x:1.14.1-SNAPSHOT
    +-com.google.cloud.bigtable:bigtable-client-core:1.14.1-SNAPSHOT
      +-com.google.cloud:google-cloud-core-http:1.93.1
        +-com.google.api-client:google-api-client:1.30.9
          +-com.google.http-client:google-http-client-jackson2:1.34.2
```